tests/ExceptionHandlerTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Fuzz\ImageResizer\Tests;
+
+use Fuzz\ImageResizer\ExceptionHandler;
+use InvalidArgumentException;
+use LogicException;
+use Mockery;
+use RuntimeException;
+
+class ExceptionHandlerTest extends ImageResizerTestCase
+{
+	/**
+	 * @runInSeparateProcess
+	 */
+	public function testItReturnsNiceErorOnInvalidArgumentException()
+	{
+		$exception = new InvalidArgumentException('Some message.');
+
+		$handled = ExceptionHandler::handleException($exception);
+
+		$this->assertEquals([
+			'error'   => 'bad_request',
+			'message' => $exception->getMessage(),
+			'trace'   => $exception->getTraceAsString(),
+		], $handled);
```

Comment:
I have mixed feelings about tests of this nature - those that assert against very thin logical layers. Ultimately, I don't question whether they have value, but rather how much value they have relative to the effort required to produce and maintain them over time.

To be clear: I don't have a leaning one way or the other, just mixed feelings.